3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -158,3 +107,5 @@ if [ "${MEMBER_ID}" = '0' ]; then
 else
   add_member "${MEMBER_HOST}"
 fi
+
+>/tmp/initialized
```

Comment:
@jim-minter Why not to do the same for replicaset too? (to allow users to update their replicaset example if they want)